ona/authentication.py

```diff
@@ -55,8 +57,15 @@ def authenticate(self, request):
         cookie_jwt = request.COOKIES.get(settings.KPI_COOKIE_NAME)
         if cookie_jwt:
             api_token = get_api_token(cookie_jwt)
-            if getattr(api_token, 'user'):
-                return api_token.user, api_token
+            user_name = api_token[0].user.username
+            user_email = api_token[0].user.email
+            # Create KPI User from Onadata user username and email
+            user = User.objects.using('default').get_or_create(
+                username=user_name, email=user_email)
```

Comment:
Based on the suggestion i've made above, we'd can go ahead and refactor the above to the following:
```python
...
user = User.objects
           .using('default')
           .get_or_create(
              username=api_token.user.username,
              email=api_token.user.email)
```